ui/text-input-edit.reel/text-input-edit.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @module ui/text-input-edit.reel
+ */
+var AbstractControl = require("montage/ui/base/abstract-control").AbstractControl,
+    KeyComposer = require("montage/composer/key-composer").KeyComposer;
+
+/**
+ * @class TextInputEdit
+ * @extends Component
+ */
+var TextInputEdit = exports.TextInputEdit = AbstractControl.specialize(/** @lends TextInputEdit# */ {
+
+    enterDocument: {
+        value: function () {
+            // this.createActionEvent();
+        }
+    },
+
+    prepareForActivationEvents: {
+        value: function () {
+            var keyboardIdentifiers = this.constructor.KEY_IDENTIFIERS,
```

Comment:
According to the way you use it (except if I missed something), KEY_IDENTIFIERS should be an Array instead of an object. It would remove the need for `Object.keys` and make the iteration simpler.